text/41_advanced_unit_testing.md

```diff
@@ -214,8 +214,8 @@ public void testHaveFunAtDuckPond() {
 
     Person p = new Person();
     DuckPond dp = Mockito.mock(DuckPond.class);
-    Mockito.verify(duckPond.haveFun(), times(1));
     p.haveFunAtDuckPond();
+    Mockito.verify(duckPond.haveFun(), times(1));
```

Comment:
I might be wrong here (I don't have an environment with JUnit and Mockito atm) but I think the verify goes after the call